tg_bot/__main__.py

```diff
@@ -428,6 +456,7 @@ def main():
 
     donate_handler = CommandHandler("donate", donate)
     migrate_handler = MessageHandler(Filters.status_update.migrate, migrate_chats)
+    is_chat_allowed_handler(MessageHandler(Filters.all & (~ Filters.private), is_chat_allowed)))
```

Comment:
so... use filters.group instead?

Reply:
This also isn't how a handler is created, why is this a function call?

Reply:
@PaulSonOfLars So long have passed since I used your fork and now I'm used to use all handlers in one line, that's why I made the mistake to make it as a function call